### DISCUSSION TITLE
QST: Shift Alaska and Hawaii to the bottom right corner of the US continental map?

### DISCUSSION THREAD

**Discussion by santinemiliano** [geopandas/geopandas#2709] [Q&A]
Just a simple is it possible to to display Alaska and Hawaii below the Continental US? similarly like plotly px.choropleth 

![Screenshot (81)](https://user-images.githubusercontent.com/76973131/121352677-18010a80-c8fb-11eb-887a-c732168ed19e.png)

**Comment by martinfleis** [comment]
You can use [`translate`](https://geopandas.readthedocs.io/en/latest/docs/reference/api/geopandas.GeoSeries.translate.html#geopandas.GeoSeries.translate) on a subset of geometries if you want to move that (not recommended as it breaks the geographic position) or plot those on separate axes as shown in [this SO answer](https://stackoverflow.com/questions/67575936/moving-alaska-and-hawaii-in-geopandas-for-choropleths).

**Comment by walkerke** [comment]
@santinemiliano consider looking at the `shift_geometry()` function in the new __pygris__ package, which is designed to do this for you: https://walker-data.com/pygris/02-geometries/
